member/src/main/proto/com/improving/app/member/domain/improving-app-member-domain.proto

```diff
@@ -0,0 +1,200 @@
+syntax =  "proto3";
+
+package com.improving.app.member.domain;
+
+import "com/improving/app/organization/domain/improving-app-organization-domain.proto";
+import "com/improving/app/tenant/domain/improving-app-tenant-domain.proto";
+import "google/protobuf/timestamp.proto";
+import "scalapb/scalapb.proto";
+
+option (scalapb.options) = {
+  no_default_values_in_constructor: true
+  preserve_unknown_fields: false
+};
+
+
+//Data Model
+message MemberId {
+  option (scalapb.message).extends = "com.improving.app.member.utils.serialize.CborSerializable";
+  string id = 1;
+}
+
+enum NotificationPreference {
+  option (scalapb.enum_options).extends = "com.improving.app.member.utils.serialize.CborSerializable";
+  NOTIFICATION_PREFERENCE_EMAIL = 0;
+  NOTIFICATION_PREFERENCE_SMS = 1;
+  NOTIFICATION_PREFERENCE_APPLICATION = 2;
+}
+
+enum MemberStatus {
+  option (scalapb.enum_options).extends = "com.improving.app.member.utils.serialize.CborSerializable";
+  MEMBER_STATUS_INITIAL = 0;
+  MEMBER_STATUS_ACTIVE = 1;
+  MEMBER_STATUS_INACTIVE = 2;
+  MEMBER_STATUS_SUSPENDED = 3;
+  MEMBER_STATUS_TERMINATED = 4;
+}
+
+message Contact {
```

Comment:
Please put this in the common project

Reply:
will align with @miguelTaningcoYoppworks and move to common

Reply:
done